frontend/encore/versioning.rst

```diff
@@ -51,6 +51,10 @@ in your ``script`` and ``link`` tags. If you're using Symfony, just activate the
             # feature is supported in Symfony 3.3 and higher
             json_manifest_path: '%kernel.project_dir%/public/build/manifest.json'
 
+.. tip::
+
+    If your project uses Symfony Flex, a file called ``assets.yaml`` is created upon installation of Webpack Encore containing the lines above, so you don't have to manually add them to ``framework.yaml``.
+
```

Comment:
What if we *assume* the user is using Flex (that's the general policy on the 4.0+ docs). We could:

A) Update the filename in code block above to be `assets.yaml`
B) Add a comment at the top of the code block:

> This file is added automatically when installing Encore with Symfony Flex

Then we can remove this extra tip :). WDYT?

Reply:
Yes you are right, that makes more sense! I've applied the changes, I also took a look at comments in other YAML files in the docs and it seems that most of them start with a lowercase letter and no blank lines between them, you think it's okay like this?